### ISSUE TITLE
Cursor does not scale

### ISSUE THREAD

**Issue by tomowad** [giowck/symphytum#109]
In Symphytum 2.5 on Kubuntu 19.10, on a 4k display scaled to 2x, the cursor changes to its unscaled dimensions, when moved into the Symphytum window.

![cursor](https://user-images.githubusercontent.com/1836512/71057297-3b922d00-212a-11ea-8fcf-b77f0e09e644.png)

**Comment by joshirio** [comment]
Probably a toolkit and desktop environment thing and there's little we can do. Maybe this is a knows issue and may get fixed by Qt/KDE.

Would be interested to know if Symphytum is the only Qt app with this issue. Another consideration: was the app installed via snap? Because snap package could cause these kind of issues, since the integration is not polished.

**Comment by tomowad** [comment]
You're right, it was installed via Snap. I hadn't realized that Discover installed some packages via that method. It is the only Qt app with which I've had this issue, but I don't have any other snaps installed.
